skill-bundles/software/manifest.ps1

```
@@ -0,0 +1,487 @@
+    "threat-modeler",
+    "security-auditor-expert",
+    "penetration-tester",
+    "code-explainer",
+    "task-planner",
+    "researcher",
+    "scout"
+)
+
+# Premium software-ai — daily skill set (~45). Full library (350+) via -FullSkills on profile installer.
+# Keeps startup light so cloud 480b has room for your project + avoids 500 errors.
+$PremiumProfileSkills = @(
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'PremiumProfileSkills' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)